go_deps.bzl

```diff
@@ -0,0 +1,4 @@
+load("@bazel_gazelle//:deps.bzl", "go_repository")
+
+def go_deps():
+  pass
```

Comment:
Copypasta from the example in the repo. What's the purpose of this, is it needed at all?

Reply:
Go dependencies are usually listed here, but you have `deps.bzl` file for this. Where to write dependencies defined [here](https://github.com/AlexTereshenkov/dg-query/blob/bazel/add-staticcheck-linters/BUILD.bazel#L31).

Reply:
I see, thanks. Removed these bits then as they are redundant.